apps/summerfi-api/lib/sdk-router-function/tsconfig.test.json

```diff
@@ -2,12 +2,14 @@
   "extends": "@summerfi/typescript-config/tsconfig.test.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "dist",
     "baseUrl": ".",
     "paths": {
-      "~src/*": ["./src/*"]
+      "~src/*": ["src/*"],
+      "@summerfi/common": ["node_modules/@summerfi/common/src/index.ts"],
```

Comment:
I think this is what you were explaining about the solution to the imports for tests. If this is going to be the way of doing it, I think we should make a proposal so every other packages uses the same approach. I fear that we have a mix of configurations at this moment and that it is still not clear which one is best

Reply:
Yes, I think this is a way to go as it turns out we cannot share path mappings and each tsconfig test needs to have own mapping as it overrides when extending base tsconfig, no way around it!
I'll refactor all existing packages to have a common approach.